src/patternfly/layouts/Flex/examples/Flex.md

```diff
@@ -769,6 +769,98 @@ Ordering - Ordering can be applied to nested <code>.pf-l-flex</code> and <code>.
 {{/l-flex}}
 ```
 
+### Row gap
```

Comment:
Could we add an example that just shows `pf-m-row-gap` on its own? That could be a simple example. Then this example could show custom row gaps.